projects/baby_driver/scripts/adc_read.py

```diff
@@ -0,0 +1,58 @@
+"""Python implementation of the adc_read function for Babydriver."""
+
+import can_util
+from gpio_port import GpioPort
+from message_defs import BabydriverMessageId
+
+NUM_PINS_PER_PORT = 16
```

Comment:
This constant should definitely go in gpio_port.py since it's generic to gpio ports in general.